_episodes/01-shell-basics.md

```diff
@@ -3,36 +3,36 @@ title: "Command line/shell"
 teaching: 150
 exercises: 30
 questions:
-- "Why and how does using the command line/shell efficiently increase reproducibility of neuroimaging studies?"
-- "How can we assure that our scripts *do the right thing*?"
+- "Why and how does using the command line/shell efficiently increase the reproducibility of neuroimaging studies?"
+- "How can we ensure that our scripts *do the right thing*?"
 objectives:
-- "Understand basic differences among available shells and their use in
+- "Understand basic differences among available shells and their use(s) in
  neuroimaging toolkits"
-- "Using a shell as a working medium"
-- "Explain the role of some of the most important environment variables"
+- "Use a shell as a working medium"
+- "Explain the roles of some of the most important environment variables"
 - "Provide hints on efficient use of the collected shell history of commands"
 - "Explain how to make shell scripts more robust and less dangerous"
 - "Introduce basics of runtime and unit testing"
-keypoints:
-- "There are a number of incompatible shells; Different
+key points:
```

Comment:
I think you must keep it as a single word since used by the system rendering this whole thing